### PR TITLE
[Attributes] Add attributes overview for 6.3

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Followup of https://github.com/symfony/symfony-docs/pull/18003 and and waiting merge of https://github.com/symfony/symfony-docs/pull/18014

Again, I'm keeping an eye on upmerge and will rebase accordingly 🙂

### CODE REVIEW

**Comment by OskarStark** [comment]
Upmerge done ✅

File: reference/attributes.rst [hunk 1]
```diff
@@ -27,11 +28,13 @@ Contracts
 Dependency Injection
 ~~~~~~~~~~~~~~~~~~~~
 
+* :class:`Symfony\\Component\\DependencyInjection\\Attribute\\AsAlias`
```
Comment (OskarStark) [inline reference/attributes.rst:31]: would be great to have an example here in a follow up PR, where we can link to

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @alexandre-daubois.

File: reference/attributes.rst (hunk 1 above)
Comment (alexandre-daubois) [reply reference/attributes.rst:31]: Yes definitely! I plan to write some doc in the next days for missing attributes :+1:

**Pull request merged**